Documentation/Scripts/fetchRefs.sh

```diff
@@ -6,43 +6,49 @@ ALLBOOKS="HTTP AQL Manual Cookbook Drivers"
 
```

Comment:
bash scripts should start with `set -u` and never with `set -e`! `set -e` breaks code in subtle ways: https://mywiki.wooledge.org/BashFAQ/105